mteb/cli.py

```diff
@@ -354,6 +368,50 @@ def add_create_meta_parser(subparsers) -> None:
     parser.set_defaults(func=create_meta)
 
 
+def add_create_table_parser(subparsers) -> None:
+    parser = subparsers.add_parser(
+        "create-table", help="Create comparison tables for MTEB models"
+    )
+
+    parser.add_argument(
+        "--results",
+        type=str,
+        default="results/",
+        help="Path to fetch results from (local folder or GitHub repo URL)",
+    )
+
+    parser.add_argument(
+        "--models",
+        type=str,
+        default=None,
+        help="Comma-separated list of models to include in the table (default: all models)",
+    )
+
+    parser.add_argument(
+        "--benchmark",
+        type=str,
+        default=None,
+        help="Benchmark to use (optional). Available benchmarks can be listed with 'mteb available_benchmarks'",
+    )
```

Comment:
I think benchmark can be also a list
```suggestion
    parser.add_argument(
        "--benchmark",
        type=str,
        nargs='*',
        choices=mteb.get_bencharls(),
        default=None,
        help="Benchmark to use (optional). Available benchmarks can be listed with 'mteb available_benchmarks'",
    )
```

Reply:
@Samoed current implementation was to support single benchmarks only. Does different benchmarks can have same tasks also or not?

Reply:
> Does different benchmarks can have same tasks also or not?.

Yes, they can.

> current implementation was to support single benchmarks only.

You can leave it as one benchmark, but it would be better to add choices.

Reply:
> > Does different benchmarks can have same tasks also or not?.
> 
> Yes, they can.
> 
So, if a particular task is present in 2 benchmarks, then does it have different implementation and results also?
If not, then I thing I can have a 1 more column in table with benchmark names. And for task present under different benchmarks, the column will list out all of them like:

| task    | benchmark | split   | subset | model1 | model2 |
|----------|----------|-------|--------|--------|
| task1   | 'MTEB(Multilingual, v1)', 'MTEB(jpn, v1)| test  | subset1  | 50.55  | 50.55  |
|         | |          | subset2  | 50.55  | 50.55  |
|          | 'MTEB(Multilingual, v1)', 'MTEB(jpn, v1)| dev  | subset1  | 50.55  | 50.55  |
|         | |          | subset2   | 50.55  | 50.55  |

Reply:
Benchmarks mean scores on tasks, so it won't have `split` and `subset`. I think you can make to select only one benchmark at a time